client/src/components/photo.jsx

```diff
@@ -2,21 +2,12 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import Modal from 'react-modal';
 
-class Photo extends React.Component {
-    constructor(props) {
-        super(props);
-        this.state = {
-            
-        }
-    }
-
-    render() {
-        return (
-            <div> 
-                <img className="photo" onMouseOver={this.props.hoverPicture} onClick={this.props.clickPicture} src={this.props.photo}/>
-            </div>
-        )
-    }
+const Photo = (props) => {
+    return (
+        <div> 
+            <img className="photo" onMouseOver={props.hoverPicture} onClick={props.clickPicture} src={props.photo}/>
+        </div>
```

Comment:
Put it through a linter please

Reply:
Will do. I'll send you the PR for it once that's done. Thanks Tom.